lib/utils/isStandardSyntaxDeclaration.js

```diff
@@ -36,6 +37,11 @@ module.exports = function (decl) {
 		return false;
 	}
 
+	// SCSS map and list declarations
+	if (value[0] === '(' && value[value.length - 1] === ')') {
+		return false;
+	}
+
```

Comment:
@sylwia-lask Thanks for the change! In addition, can you add a test case to `lib/utils/__tests__/isStandardSyntaxDeclaration.test.js`, please?